Sample/Cocoapods/Podfile.lock

```diff
@@ -0,0 +1,67 @@
+PODS:
```

Comment:
This shouldn't be here. It's the v2 Sample app. The v3 Sample doesn't need any dependencies so far.